server.rb

```diff
@@ -56,7 +63,7 @@ def get_message
   post "/suggest_song" do
     # enter Artist, Title, Album=nil
     # submit and save to Songs table
-    # require_current_user
+    require_current_user
```

Comment:
I wanted that to say "require user" just like I have in line 99. I believe auto correct may have happened there.